src/ec/packages/ec-component-button/__snapshots__/button.test.js.snap

```diff
@@ -1,87 +1,147 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EC - Button CTA button - icon after renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
-    <span class=\\"ecl-button__label\\">CTA Button with icon</span>
-
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
+    class="ecl-button__container"
+  >
+    <span
+      class="ecl-button__label"
+    >
+      CTA Button with icon
+    </span>
     <svg
-      class=\\"ecl-icon ecl-icon--fluid ecl-button__icon ecl-button__icon--after\\"
-      focusable=\\"false\\"
-      aria-hidden=\\"true\\"
+      aria-hidden="true"
+      class="ecl-icon ecl-icon--fluid ecl-button__icon ecl-button__icon--after"
+      focusable="false"
     >
-      <use xlink:href=\\"static/icons.svg#ui--corner-arrow\\"></use>
+      <use
+        xlink:href="static/icons.svg#ui--corner-arrow"
+      />
     </svg>
   </span>
 </button>
-"
 `;
 
 exports[`EC - Button CTA button - icon before renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
+    class="ecl-button__container"
+  >
     <svg
-      class=\\"ecl-icon ecl-icon--fluid ecl-icon--rotate-90 ecl-button__icon ecl-button__icon--before\\"
-      focusable=\\"false\\"
-      aria-hidden=\\"true\\"
+      aria-hidden="true"
+      class="ecl-icon ecl-icon--fluid ecl-icon--rotate-90 ecl-button__icon ecl-button__icon--before"
```

Comment:
if we want to compare the snapshots with a reference — coming from the specs —, then we have to take care of the order of the CSS classes 🤔

Reply:
Question is: is it important enough to enforce order of classes?

Reply:
the order of the CSS classes in HTML does **not** matter, and that's the issue here: if the specs say `class="ecl-icon ecl-icon--fluid ecl-icon--rotate-90"` and you have `
class="ecl-icon ecl-icon--rotate-90 ecl-icon--fluid`, jest will think there's an error...